README.md

```diff
@@ -1,3 +1,20 @@
 # Books-API
 
 An api for managing books, book clubs, and their members.
+
+## Start the API
+
+Use the following command to start the API:
+
+```go
+go run ./cmd/main.go
+```
+
+The API should be running on port `8080`.
+
+Or use the start script:
+
+```bash
+chmod +x ./start.sh
```

Comment:
If you chmod it before committing it, won't it be marked as an executable?

Reply:
I'm not too sure. I just make sure to include in case we forget :D